imagekit-sdk/src/test/java/io/imagekit/sdk/ImageKitTest.java

```diff
@@ -110,6 +107,34 @@ public void getUrl_with_slash_in_path() {
         assertThat(SUT.getConfig().getUrlEndpoint()+"tr:w-400,h-600/default-image.jpg?ik-sdk-version="+ Version.VERSION_CODE,is(url));
     }
 
+    @Test
+    public void getUrl_with_full_url_in_path_with_signature() {
+        List<Map<String, String>> transformation=new ArrayList<Map<String, String>>();
+        Map<String,Object> options=new HashMap<>();
+        options.put("path","http://placehold.it/1024x1025?text=test");
+        options.put("transformation",transformation);
+        options.put("signed", true);
+        options.put("expireSeconds", 1000);
+        String url=SUT.getUrl(options);
+        assertSignedUrl(SUT.getConfig().getUrlEndpoint() + "http%3A%2F%2Fplacehold.it%2F1024x1025%3Ftext%3Dtest", url);
+    }
+
+    @Test
+    public void getUrl_with_full_url_in_path_with_transform_with_signature() {
```

Comment:
Should there be a test with a fallback URL too?

Reply:
Added. 
They didn't have a single test to test the defaultImage functionality anywhere else.
👍